integration-test/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.docs;
+
+import org.ehcache.CacheManager;
+import org.junit.Test;
+
+import static org.ehcache.config.builders.CacheConfigurationBuilder.newCacheConfigurationBuilder;
+import static org.ehcache.config.builders.CacheManagerBuilder.newCacheManagerBuilder;
+import static org.ehcache.config.builders.ResourcePoolsBuilder.heap;
+
+/**
+ * Samples to get started with Ehcache 3. Java 8 syntax
+ */
+@SuppressWarnings("unused")
+public class GettingStarted {
+
+  @Test
+  public void cachemanagerExample() {
+    // tag::java8Example[]
+    try(CacheManager cacheManager = newCacheManagerBuilder() // <1>
```

Comment:
Huh? That's because you have in mind that we never open and close like that. But in fact we do once in a while and don't want to mess with the `UserManagedCached` and its missing services. But I agree that the example isn't showing much so I can remove it.